Dockerfile

```diff
@@ -3,11 +3,14 @@ MAINTAINER Remi Hakim @remh
 RUN rm /bin/sh && ln -s /bin/bash /bin/sh
 
 RUN apt-get update && apt-get install -y \
-    git \
     curl \
     procps \
     fakeroot
 
+RUN echo "deb http://http.debian.net/debian squeeze-backports main" >/etc/apt/sources.list.d/squeeze-backports.list
```

Comment:
shouldn't you add that command before the previous `apt-get update` ? 
This way you can just run one apt-get install command ?